pkg/templates/pdbminavailable/template.go

```diff
@@ -15,6 +15,7 @@ import (
 	"golang.stackrox.io/kube-linter/pkg/objectkinds"
 	"golang.stackrox.io/kube-linter/pkg/templates"
 	"golang.stackrox.io/kube-linter/pkg/templates/pdbminavailable/internal/params"
+	autoscalingV2 "k8s.io/api/autoscaling/v2"
```

Comment:
We need to support all versions :( 
There is a method in extract package that could be useful here although I think we need another one to extract target ref name

https://github.com/stackrox/kube-linter/blob/809459200f8a4f2a4cfbf4f9598090ab73bbeeec/pkg/extract/hpa_spec.go#L11-L25

```go
// HPAScaleTargetRefName extracts Spec.ScaleTargetRef.Name
func HPAScaleTargetRefName(obj k8sutil.Object) (string, bool) {
	switch hpa := obj.(type) {
	case *autoscalingV2Beta1.HorizontalPodAutoscaler:
		return hpa.Spec.ScaleTargetRef.Name, true
	case *autoscalingV2Beta2.HorizontalPodAutoscaler:
		return hpa.Spec.ScaleTargetRef.Name, true
	case *autoscalingV2.HorizontalPodAutoscaler:
		return hpa.Spec.ScaleTargetRef.Name, true
	case *autoscalingV1.HorizontalPodAutoscaler:
		return hpa.Spec.ScaleTargetRef.Name, true
	default:
		return "", false
	}
}
```